docs/content/about.md

```diff
@@ -107,6 +107,57 @@ public class MySqlBinlogSourceExample {
   }
 }
 ```
+The following JSON shows the basic CDC events. 
+
+```json
+{
+  "before": {
+    "id": 111,
+    "name": "scooter",
+    "description": "Big 2-wheel scooter",
+    "weight": 5.18
+  },
+  "after": {
+    "id": 111,
+    "name": "scooter",
+    "description": "Big 2-wheel scooter",
+    "weight": 5.15
+  },
+  "source": {...},
+  "op": "u",
+  "ts_ms": 1589362330904,
+  "transaction": null
+}
+```
+Note: please refer to [Debezium documentation](https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-events
+)  about the meaning of each fields.
+
+In some cases, users can use the `JsonDebeziumDeserializationSchema(true)` Constructor to enabled include schema in the message. Then the Debezium JSON message may look like this:
+```json
+{
+  "schema": {...},
+  "payload": {
+    "before": {
+      "id": 111,
+      "name": "scooter",
+      "description": "Big 2-wheel scooter",
+      "weight": 5.18
+    },
+    "after": {
+      "id": 111,
+      "name": "scooter",
+      "description": "Big 2-wheel scooter",
+      "weight": 5.15
+    },
+    "source": {...},
+    "op": "u",
+    "ts_ms": 1589362330904,
+    "transaction": null
+  }
+}
+```
+Usually, this is not recommended to include schema because this makes the messages very verbose and reduces parsing performance.
```

Comment:
```suggestion
Usually, it is recommended to exclude schema because schema fields makes the messages very verbose which reduces parsing performance.
```